### ISSUE TITLE
Issues with  label encoding and validation loss

### ISSUE THREAD

**Issue by nisnab** [jiangqy/Customized-DataLoader#3]
I have some problems with  making data loaders(especially  encoding labels).
Can you please assist.
